github_issue_summarization/ks-kubeflow/components/params.libsonnet

```diff
@@ -55,15 +55,15 @@
       githubToken: "",
     },
     "tfjob-v1alpha2": {
-      name: "tfjob-v1alpha2",
-      image: "gcr.io/kubeflow-dev/tf-job-issue-summarization:v20180425-e79f888",
+      name: "tfjob-issue-summarization",
+      image: "gcr.io/kubeflow-examples/tf-job-issue-summarization:latest",
```

Comment:
Perhaps we should re-tag so as to not break this tomorrow when somebody updates the image?

Reply:
+1 Its best to pin to a specific image.

Reply:
Fair point - just pushed a new image & updated the tag